packages/dx-react-core/src/ref-holder.tsx

```diff
@@ -7,15 +7,16 @@ export const RefHolder = React.forwardRef((
 ) => {
   return React.isValidElement(children) ?
     typeof children.type === 'string'
-      ? React.cloneElement(children, { ref })
-      : React.cloneElement(children, { forwardedRef: children.props?.forwardedRef ? (node) => {
-        children.props.forwardedRef(node);
-        if (typeof ref === 'function') {
-          ref(node);
-        } else if (ref) {
-          ref.current = node;
-        }
-      } : ref }) :
+      ? React.cloneElement(children as React.ReactElement<any>, { ref })
```

Comment:
Is it possible to replcae
 children: React.ReactNode 
by 
children: React.ReactElement<any>